runtime/opt/zalando/runtime/Docker

```diff
@@ -29,6 +30,15 @@ def get_env_options(config: dict):
         yield '-e'
         yield 'ETCD_URL=http://172.17.42.1:2379'
 
+    yield '-e'
+    yield 'APPLICATION_ID={}'.format(config.get('application_id'))
+
+    yield '-e'
+    yield 'APPLICATION_VERSION={}'.format(config.get('application_version'))
+
+    yield '-e'
+    yield 'INSTANCE_ID={}'.format(boto.utils.get_instance_metadata()['instance-id'])
```

Comment:
We could add this as an optional value if you need it and we don't want to
vendor lock taupage.
Am 04.06.2015 07:54 schrieb "André Hartmann" <notifications@github.com>:

> In runtime/opt/zalando/runtime/Docker
> <https://github.com/zalando-stups/taupage/pull/47#discussion_r31694928>:
>
> > @@ -29,6 +30,15 @@ def get_env_options(config: dict):
> >          yield '-e'
> >          yield 'ETCD_URL=http://172.17.42.1:2379'
> >
> > +    yield '-e'
> > +    yield 'APPLICATION_ID={}'.format(config.get('application_id'))
> > +
> > +    yield '-e'
> > +    yield 'APPLICATION_VERSION={}'.format(config.get('application_version'))
> > +
> > +    yield '-e'
> > +    yield 'INSTANCE_ID={}'.format(boto.utils.get_instance_metadata()['instance-id'])
>
> I'm using these information in zalando-stups/friboo#15
> <https://github.com/zalando-stups/friboo/pull/15> to build unique log
> file names.
>
> I think it's useful for many applications to know about their environment.
> "INSTANCE_ID" could also have been called "HOST_ID" or whatever. If we do
> not pass the information to the docker process, but use the EC2 metadata
> service from inside the application, we would bind all of our apps to AWS.
> This would IMHO make the vendor locking even worse.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando-stups/taupage/pull/47/files#r31694928>.
>